modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/variablereport/v0.1.2.json

```diff
@@ -0,0 +1,50 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/variablereport/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:VariableReportShape",
+      "@type": "sh:NodeShape",
+      "label": "Validation report shape",
+      "targetClass": "nsg:VariableReport",
```

Comment:
I suggest to increase the major version as the target class has changed.

Reply:
Done.